### PR TITLE
Add a note that migration from Aligned with "book tokens" to Block is not supported

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2021.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-2021.surge.sh/operations-manual/2025.01/database-internals/store-formats)

File: modules/ROOT/pages/database-internals/store-formats.adoc [hunk 1]
```diff
@@ -118,12 +118,19 @@ bin/neo4j-admin database copy --to-format="block" mydb blockdb
...
 === Changing the store format of existing databases
 
-Starting from 5.23, `block` format is the preferred format for Enterprise Edition due to its superior performance and scalability.
-Therefore, migrating all databases to `block` format is recommended to ensure optimal performance.
+Starting from 5.23, `block` format is the recommended format for Enterprise Edition due to its superior performance and scalability.
+It uses advanced data structures and inlining techniques to enhance data locality, which leads to better resource utilization.
+Therefore, it is highly-recommended that Enterprise Edition users migrate all databases to `block` format at their earliest convenience to ensure the best possible performance.
+`block` format is the default format for new databases created in 5.23 and later, and is the default format for all databases in 5.26 and later.
+
+From 5.25 onwards, `block` format supports token names (including label, property key, and relationship type names) of any length up to the GQL identifier max length of 16,383 characters.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-internals/store-formats.adoc:129]:
```suggestion
From 5.25 onwards, the `block` format supports token names (including node labels, property keys, and relationship type names) of any length up to the GQL identifier max length of 16,383 characters.
```

File: modules/ROOT/pages/database-internals/store-formats.adoc [hunk 2]
```diff
@@ -118,12 +118,19 @@ bin/neo4j-admin database copy --to-format="block" mydb blockdb
...
 
-
 [[change-store-format]]
 === Changing the store format of existing databases
 
-Starting from 5.23, `block` format is the preferred format for Enterprise Edition due to its superior performance and scalability.
-Therefore, migrating all databases to `block` format is recommended to ensure optimal performance.
+Starting from 5.23, `block` format is the recommended format for Enterprise Edition due to its superior performance and scalability.
+It uses advanced data structures and inlining techniques to enhance data locality, which leads to better resource utilization.
+Therefore, it is highly-recommended that Enterprise Edition users migrate all databases to `block` format at their earliest convenience to ensure the best possible performance.
```
Comment (NataliaIvakina) [inline modules/ROOT/pages/database-internals/store-formats.adoc:126]:
```suggestion
Therefore, it is highly recommended that Enterprise Edition users migrate all databases to `block` format at their earliest convenience to ensure the best possible performance.
```

Comment (NataliaIvakina) [reply modules/ROOT/pages/database-internals/store-formats.adoc:126]: It seems that 'highly recommended' is not an adjective here.

**Review by NataliaIvakina — APPROVED** [review]
Looks good!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2021.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-2021.surge.sh/operations-manual/2025.01/database-internals/store-formats)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-2021.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-2021.surge.sh/operations-manual/2025.01/database-internals/store-formats)

**Pull request merged**